containers/kubernetes/how-to/upgrade-kubernetes-version.mdx

```diff
@@ -32,11 +37,124 @@ categories:
+Complete the following steps to do so:
+1. Create the pool:
+    ```sh
+    scw k8s pool create container-runtime=containerd zone=$POOL_ZONE size=$SIZE_OF_YOUR_OLD_POOL version=$YOUR_CLUSTER_VERSION cluster-id=$CLUSTER_ID
+    ```
+2. Wait for the nodes to be provisioned:
+    ```sh
+    scw k8s pool wait $POOL_ID
+    ```
+3. In parallel, you can start cordoning nodes using the old runtime. This way, the workload will get rescheduled directly on the `containerd` nodes.
+    ```sh
+    kubectl cordon $OLD_NODE_A $OLD_NODE_B $OLD_NODE_C ...
+    ```
+4. Once the new nodes are ready, you can start draining the old nodes:
+    <Message type="note">
+      You may need to add the option `--delete-emptydir-data` if you used local disk as a scratchpad.
+    </Message>
+    ```sh
+    kubectl drain --ignore-daemonsets $NODE_TO_DRAIN
+    ```
+    <Message type="important">
+      **Do not** drain all the nodes simultaneously. Make sure to do it sequentially, while checking whether your workload is behaving as expected.
+    </Message>
+5. At this point, nodes from your old pool should be emptied of any workload. If so, they are ready for deletion:
+    ```sh
+    kubectl delete node $OLD_NODE_A $OLD_NODE_B $OLD_NODE_C ...
+    ```
+6. Delete your old pool:
+    ```sh
+    scw k8s pool delete $OLD_POOL_ID
+    ```
+
+### CNIs
+
+If your cluster is using a deprecated CNI, and you are willing to upgrade to a newer k8s version, you will most likely have to spin up a new Kapsule cluster.
+As of now, we do not provide an easy way to change CNI. This component is integrated within each cluster node. This results in a tricky transition.
+For more help, check out the following resources:
+- The #k8s channel on our [Slack community](https://scaleway-community.slack.com)
+- Our [support ticketing system](/console/my-account/how-to/open-a-support-ticket/)
+
+### Effective upgrade
+
+From here, two options are available: you are either upgrading **one minor version** or **multiple ones**.
+
+#### One minor version
+
+This option is the most straightforward and requires you to first upgrade your control-plane.
+```sh
+scw k8s cluster upgrade $CLUSTER_ID version=$NEW_K8S_VERSION
+```
+
+<Message type="tip">
+  You can also upgrade the pools by appending the previous command with `upgrade-pools=true`.
+</Message>
+
+Additionally, you can upgrade one pool independently by running the following command:
+```sh
+scw k8s pool upgrade $POOL_ID version=$NEW_K8S_VERSION
+```
+
+If you wish to migrate your workload manually, you can do so by following the steps described in the [runtimes](#container-runtimes) section.
+<Message type="important">
+  Make sure to adapt the pool creation step.
+</Message>
+```sh
+scw k8s pool create zone=$OLD_POOL_ZONE size=$SIZE_OF_YOUR_OLD_POOL version=$NEW_CLUSTER_VERSION cluster-id=$CLUSTER_ID
+```
+
+Monitor your workload, congrats you are using a newer, maybe the latest Kubernetes version.
```

Comment:
This line seems weird, maybe it could be deleted?

Reply:
```suggestion
```